File: main/src/main/java/net/fwitz/math/main/binary/split/functions/trig/SplitTanZ.java
package net.fwitz.math.main.binary.split.functions.trig;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitTanZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("tan(z)")
                .computeFn(SplitComplex::tan)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/trig/SplitTanZ.kt
package net.fwitz.math.main.binary.split.functions.trig

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitTanZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("tan(z)")
        .computeFn(SplitComplex::tan)
        .render()
}